eisbuk-admin/public/locales/en/translation.json

```diff
@@ -63,6 +67,7 @@
     "AddAthlete": "AddAthlete"
   },
   "DashboardPage": {
+    "Reservations": "Reservations",
     "Bookings": "Bookings"
```

Comment:
The both reservations and bookings are unnecessarry (in `/it` translation file they're both "prenotazioni"). The bookings is more consistent IMO (since we have `BookingsByDay`, not `ReservationsByDay`)